package.json

```diff
@@ -68,7 +68,9 @@
     "url": "git://github.com/standard/standard.git"
   },
   "scripts": {
-    "test": "./bin/cmd.js --verbose && tape test/*.js",
+    "test": "npm run test-internal && npm run test-external",
+    "test-internal": "./bin/cmd.js --verbose && tape test/internal/*.js",
```

Comment:
Maybe we can keep the internal ones in the top folder? And have them be `test/*.js`? Less verbose?

Reply:
Yes, it is less verbose! :+1: 
Done. :smile: